client/overview.jsx

```diff
@@ -1,15 +1,29 @@
 class Overview extends React.Component {
```

Comment:
Ran your code! Getting one error on client out of the gate = 'Cannot GET /favicon.ico:1'. Are you trying to load a favicon?

Reply:
I don't think so because i dont know what that is... I will keep a look out for it though.